gym/envs/registration.py

```diff
@@ -78,7 +78,14 @@ def make(self, **kwargs):
         spec = copy.deepcopy(self)
         spec._kwargs = _kwargs
         env.unwrapped.spec = spec
+        if env.spec.max_episode_steps is not None:
+            from gym.wrappers.time_limit import TimeLimit
+
+            env = TimeLimit(env, max_episode_steps=env.spec.max_episode_steps)
+        else:
```

Comment:
There should be some way to disable wrapping the environment with the OrderEnforcing wrapper via the env.spec.